package.json

```diff
@@ -1,16 +1,22 @@
 {
-  "name": "express-authorshaven",
+  "name": "express-barefoot-nomad",
   "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
+  "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon --exec node -r dotenv/config ./src/index.js",
-    "build": "node -r dotenv/config ./src/index.js"
+    "production": "babel-node ./src/index.js",
```

Comment:
`babel-node` can only be used for development.

Reply:
ooh so I create a build file for production?

Reply:
I feel we should have two deployments for `master` and `develop` branches.

Reply:
There are two deployments already, one for [staging](https://boondocks-bn-backend-staging.herokuapp.com) which I think should be on develop branch and then the other one for [production](https://boondocks-bn-backend.herokuapp.com/) which I should be on master  branch